org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/highlight/HighlightReconcilingStrategy.java

```diff
@@ -285,7 +289,10 @@ void removeOccurrenceAnnotations() {
 		}
 	}
 
-	private String kindToAnnotationType(DocumentHighlightKind kind) {
+	private String kindToAnnotationType(@Nullable DocumentHighlightKind kind) {
+		if (kind == null)

```

Comment:
is this specified by the protocol?

Reply:
Yes.

![image](https://github.com/eclipse/lsp4e/assets/426959/fbfcf1f5-9c49-4405-93e8-97d84c6bdb3d)